README.md

```diff
@@ -141,6 +141,40 @@ Some examples:
 ./ankerctl.py mqtt rename-printer BoatyMcBoatFace
 ```
 
+
+## Docker
+
+While running the python script is generally prefered, there may be situations where you want a more portable solution. For this, a docker image is provided.
+
+```sh
+docker build -t ankerctl . --no-cache
```

Comment:
Why do we use `--no-cache` here?

What does it do, exactly?

Reply:
Ah, that's actually reasonable. And thanks for the explanation :)

But yes, I think we should get rid of it for the readme, to make the command a little bit simpler for beginners.